### PR TITLE
fix #70 Name only parameters are ignored by parseParams

### LINKED ISSUE #70: Name only parameters are ignored by parseParams
Hi 

I noticed that in ST2110-20 the `interlace` parameter is a standalone declaration and do not have a value.
(See in this validation library: https://github.com/Streampunk/sdpoker/blob/b56fc0e0f5c0fd00a0696aac45795db2a660ea43/checkST2110.js#L607 )

In this case the parameter is omitted from the result structure.
From my investigation this  due to this this condition : https://github.com/clux/sdp-transform/blob/17386b7ea7a8ac917156c4af5a500503905a989b/lib/parser.js#L68

### PR DESCRIPTION
**Pull request by manuc66**

### CODE REVIEW

**Comment by clux** [comment]
Ooh. Thanks a lot! Hadn't really considered anything but the base level use of `parseParams`, as it was only added as a kind of ad-hoc helper.

This, and the good tests does demonstrate the case perfectly though. Will merge and make a patch release :+1:

File: test/parse.test.js [hunk 1]
```diff
@@ -637,3 +637,36 @@ test('ST2022-6', function *(t) {
...
+  var video = media[0];
+  var sourceFilter = video.sourceFilter;
+  t.equal(sourceFilter.filterMode, 'incl', 'filter-mode is "incl"');
+  t.equal(sourceFilter.netType, 'IN', 'nettype is "IN"');
+  t.equal(sourceFilter.addressTypes, 'IP4', 'address-type is "IP4"');
+  t.equal(sourceFilter.destAddress, '239.100.9.10', 'dest-address is "239.100.9.10"');
+  t.equal(sourceFilter.srcList, '192.168.100.2', 'src-list is "192.168.100.2"');
+
+  t.equal(video.type, 'video', 'video type');
+  console.log(video);
```
Comment (manuc66) [inline test/parse.test.js]: I should had removed this line...

File: test/st2110-20.sdp [hunk 2]
```diff
@@ -0,0 +1,23 @@
+`v=0
```
Comment (ibc) [inline test/st2110-20.sdp]: This is wrong. SDP files must not have ` at the beginning and at the end.

Comment (clux) [reply test/st2110-20.sdp]: ah thanks, i removed one of those stray ones, killed the second one now :-)

File: test/st2110-20.sdp [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+o=- 123456 11 IN IP4 192.168.100.2
+s=Example of a SMPTE ST2110-20 signal
+i=this example is for 720p interlaced video
+t=0 0
+a=recvonly
+a=group:DUP primary secondary
+m=video 50000 RTP/AVP 112
+c=IN IP4 239.100.9.10/32
+a=source-filter:incl IN IP4 239.100.9.10 192.168.100.2
```
Comment (ibc) [inline test/st2110-20.sdp]: This line `a=source-filter:incl IN IP4 239.100.9.10 192.168.100.2` is invalid. According to the corresponding [ABNF](https://tools.ietf.org/html/rfc4570#section-3.2.4), there must be a `SP` (space) before `incl`:

**Pull request merged**